### PR TITLE
Allows flag_name to pass thru to the coveralls.io API

### PR DESCRIPTION
**Pull request by jekku**
## What's here?

We think flag_name is a nice addition to the payload, given that it's a very good organisation tool to sort out your coveralls in groups. Or just get them tagged with anything you want in general for grouping purposes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,7 @@ Usage: mix coveralls <Options>
     -o (--output-dir)   Write coverage information to output dir.
     -u (--umbrella)     Show overall coverage for umbrella project.
     -v (--verbose)      Show json string for posting.
+    --flagname          coveralls.io 'flag_name' option (See coveralls.io API Reference)
```
Comment (parroty) [inline README.md:136]: I assume the `coveralls.io API Reference` is about the following, but I am wondering that the meaning of the option is not clear enough.
- https://docs.coveralls.io/api-reference

How do you think about adding description like `Job flag name which will be shown in the Coveralls UI`? (if it makes sense)

- https://github.com/marketplace/actions/coveralls-github-action

**Review by parroty — COMMENTED** [review]
Hi. Thank you for the PR :bow:. Could let let me ask your opinion about the description of the option?

**Comment by parroty** [comment]
Sorry, I merged `#289` first, I appreciate if you could check the conflict too 🙇 .

File: README.md (hunk 1 above)
Comment (jekku) [reply README.md:136]: Makes sense :) Will be sorting this out today.

**Comment by jekku** [comment]
Sorted the feedback and fixed the merge conflicts. :) Thanks again for taking a look

**Comment by parroty** [comment]
Thank you!

**Pull request merged**